geometry_msgs/msg/VelocityStamped

```diff
@@ -0,0 +1,10 @@
+# This expresses the timestamped velocity of a frame 'body_frame_id' in the reference frame 'header.frame_id' expressed from arbitrary observation pose 'observation_pose'.
+# The message is often used in simplified versions where:
+# - observation pose is not defined, i.e., it is assumed that the observation pose is the origin of the 'header.frame_id';
```

Comment:
I'd add a bullet: If the observation pose is an established frame, you may set its frame_id only in the pose (since the relative pose to that frame is Identity). It would be good to make folks note how to use this if everything at hand is a TF frame

Reply:
```suggestion
# - observation pose is not defined, i.e., it is assumed that the observation pose is the origin of the 'header.frame_id';
# - if the observation pose is an established frame, you may set its frame_id only in the pose (since the relative pose to that frame is Identity);
```